perception/image_projection_based_fusion/src/pointpainting_fusion/preprocess_kernel.cu

```diff
@@ -84,10 +85,10 @@ __global__ void generateFeatures_kernel(
     pillarSumSM[threadIdx.x] = {0, 0, 0};
   }
 
-  for (std::size_t i = 0; i < POINT_FEATURE_SIZE; ++i) {
+  for (std::size_t i = 0; i < encoder_in_feature_size - 5; ++i) {
     pillarSM[pillar_idx_inBlock][point_idx][i] = voxel_features
-      [POINT_FEATURE_SIZE * pillar_idx * MAX_POINT_IN_VOXEL_SIZE + POINT_FEATURE_SIZE * point_idx +
-       i];
+      [(encoder_in_feature_size - 5) * pillar_idx * MAX_POINT_IN_VOXEL_SIZE +
+       (encoder_in_feature_size - 5) * point_idx + i];
```

Comment:
Cloud you use any variable name to this magic number `5` ?

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/4314/commits/80de05a8fe4609684f12f795462e73418781f0c2#diff-fcfadbc8f49e709ac4855aae2845da23283ade00023e7825aa155cabec369dfcR88 and https://github.com/autowarefoundation/autoware.universe/pull/4314/commits/a5c023f2b35dce04121d9ea75535189999e8bf0d#diff-fcfadbc8f49e709ac4855aae2845da23283ade00023e7825aa155cabec369dfcR127